crates/runtime/src/dataconnector/github/mod.rs

```diff
@@ -441,7 +450,22 @@ impl std::str::FromStr for GitHubQueryMode {
     }
 }
 
+fn warn_if_provided(
+    parameters: Vec<(&str, bool)>,
+    table_type: &str,
+    connector_component: &ConnectorComponent,
+) {
+    for (param, present) in parameters {
+        if present {
+            tracing::warn!(
+                "Parameter '{param}' is not supported for the '{table_type}' table type in {connector_component} and will be ignored."
```

Comment:
```suggestion
                "The parameter '{param}' is not supported for the {connector_component}, as a '{table_type}' table. For details, visit: https://spiceai.org/docs/components/data-connectors/github"
```

E.g.

`The parameter 'include_comments' is not supported for the dataset blah, as a 'commits' table. For details, visit: https://spiceai.org/docs/components/data-connectors/github`